t/math/declare.xml

```diff
@@ -46,51 +46,51 @@
             <XMDual>
               <XMApp>
                 <XMTok meaning="list"/>
-                <XMRef idref="S1.Ex1.m1.1"/>
-                <XMRef idref="S1.Ex1.m1.2"/>
-                <XMRef idref="S1.Ex1.m1.3"/>
                 <XMRef idref="S1.Ex1.m1.4"/>
                 <XMRef idref="S1.Ex1.m1.5"/>
                 <XMRef idref="S1.Ex1.m1.6"/>
                 <XMRef idref="S1.Ex1.m1.7"/>
                 <XMRef idref="S1.Ex1.m1.8"/>
+                <XMRef idref="S1.Ex1.m1.9"/>
+                <XMRef idref="S1.Ex1.m1.10"/>
+                <XMRef idref="S1.Ex1.m1.11"/>
               </XMApp>
               <XMWrap>
-                <XMTok decl_id="S1.XMD4" font="italic" role="ID" xml:id="S1.Ex1.m1.1">x</XMTok>
+                <XMTok decl_id="S1.XMD4" font="italic" role="ID" xml:id="S1.Ex1.m1.4">x</XMTok>
                 <XMTok role="PUNCT">,</XMTok>
-                <XMApp decl_id="S1.XMD5" role="ID" xml:id="S1.Ex1.m1.2">
+                <XMApp decl_id="S1.XMD5" role="ID" xml:id="S1.Ex1.m1.5">
                   <XMTok role="SUPERSCRIPTOP" scriptpos="post1"/>
                   <XMTok font="italic" role="UNKNOWN">x</XMTok>
                   <XMTok fontsize="70%" name="prime" role="SUPOP">′</XMTok>
                 </XMApp>
                 <XMTok role="PUNCT">,</XMTok>
-                <XMApp decl_id="S1.XMD7" role="ID" xml:id="S1.Ex1.m1.3">
+                <XMApp decl_id="S1.XMD7" role="ID" xml:id="S1.Ex1.m1.6">
                   <XMTok role="SUBSCRIPTOP" scriptpos="post1"/>
                   <XMTok font="italic" role="UNKNOWN">x</XMTok>
                   <XMTok fontsize="70%" meaning="1" role="NUMBER">1</XMTok>
                 </XMApp>
                 <XMTok role="PUNCT">,</XMTok>
-                <XMDual role="ID" xml:id="S1.Ex1.m1.4">
+                <XMDual role="ID" xml:id="S1.Ex1.m1.7">
                   <XMApp>
                     <XMTok decl_id="S1.XMD6"/>
-                    <XMRef idref="XM41"/>
+                    <XMRef idref="S1.Ex1.m1.1"/>
```

Comment:
Ah, I see, very nice test, `n` gets reused in a number of locations in the same top-level formula.